dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/TrackerController.java

```diff
@@ -80,50 +86,77 @@
 
     private final Notifier notifier;
 
-    public TrackerController(
-        TrackerImportService trackerImportService,
-        RenderService renderService,
-        ContextService contextService,
-        TrackerMessageManager trackerMessageManager,
-        Notifier notifier )
-    {
-        this.trackerImportService = trackerImportService;
-        this.renderService = renderService;
-        this.contextService = contextService;
-        this.trackerMessageManager = trackerMessageManager;
-        this.notifier = notifier;
-    }
-
     @PostMapping( value = "", consumes = MediaType.APPLICATION_JSON_VALUE )
-//    @PreAuthorize( "hasRole('ALL') or hasRole('F_TRACKER_IMPORTER_EXPERIMENTAL')" )
-    public void postJsonTracker( HttpServletRequest request, HttpServletResponse response, User currentUser )
+    // @PreAuthorize( "hasRole('ALL') or hasRole('F_TRACKER_IMPORTER_EXPERIMENTAL')" )
```

Comment:
Can we remove this if this is no longer required?

Reply:
I'm not sure, async endpoint still have this one (commented). SYSTEM user have this permission so maybe we might even keep it still uncommented, or remove at all if not necessary.